app/models/links/my_campus_links.rb

```diff
@@ -9,6 +9,28 @@ module Links
   # - RailsAdmin comes with an optional History feature to track who changed what, but it's disabled here.
   class MyCampusLinks
     def get_feed
+      links_json = campus_links_json
+      load_cs_link_api_entries(links_json)
+    end
+
+    def load_cs_link_api_entries(links_json)
```

Comment:
This brings in the ability for us to define CS Link API entries for our "My Campus" page links.